File: src/main/java/net/oxyopia/vice/config/features/worlds/ShowdownStorage.java
package net.oxyopia.vice.config.features.worlds;

import com.google.gson.annotations.Expose;
import net.oxyopia.vice.data.gui.Position;

public class ShowdownStorage {

	@Expose
	public long lastKnownTrainSpawn = -1L;

	@Expose
	public Position trainTimerPos = new Position(175f, 150f);

}

File: src/main/java/net/oxyopia/vice/config/features/worlds/ShowdownStorage.kt
package net.oxyopia.vice.config.features.worlds

import com.google.gson.annotations.Expose
import net.oxyopia.vice.data.gui.Position

class ShowdownStorage {

	@Expose
	var lastKnownTrainSpawn: Long = -1L

	@Expose
	var trainTimerPos: Position = Position(175f, 150f)

}